wordpress-mobile/WordPress-iOS
# Theme Browser: replace No Results view

Fixes #9729 

**To test:**
- Go to Site > Themes > Search.
- Enter a search term that returns no results.
- Verify the view is:
![new_nrv](https://user-images.githubusercontent.com/1816888/42483641-efdd58de-83ab-11e8-926c-b7366c948329.png)


**NOTE:**
There is a fair amount of code duplicated when showing the `NoResultsViewController` (i.e. other controllers have almost identical `setupNoResultsViewController`, `showNoResults`, `hideNoResults` methods). I'll be addressing that in a future PR.
